app/view_models/admin/auction_show_view_model.rb

```diff
@@ -15,7 +15,7 @@ def csv_report_partial
   end
 
   def c2_status_partial
-    if auction.purchase_card == 'default' && auction.c2_status != 'approved'
+    if auction.purchase_card == 'default' && auction.c2_status != 'approved' || auction.pending_acceptance?
```

Comment:
I know it's not required, but I think I would like a () around the `auction.c2_status != 'approved' || auction.pending_acceptance?`